crates/router/src/connector/stripe/transformers.rs

```diff
@@ -426,6 +443,26 @@ pub enum StripeBankNames {
     SnsBank,
     TriodosBank,
     VanLanschot,
+    PlusBank,
+    EtransferPocztowy24,
+    BankiSpbdzielcze,
+    BankNowyBfgSa,
+    GetinBank,
+    Blik,
+    NoblePay,
+    #[serde(rename = "idea_bank")]
```

Comment:
Since there's a `#[serde(rename_all = "snake_case")]`, this rename is not needed.